src/ploigos_step_runner/step_implementers/create_container_image/buildah.py

```diff
@@ -28,12 +28,14 @@
 --------------------------|------------
 `container-image-version` | Container version to tag built image with
 `image-tar-file`          | Path to the built container image as a tar file
+`image-tar-hash`          | Path to the built container image as a tar file
```

Comment:
why does this need to be an ouptut here? why not if someone needs a hash, they calcualate it themselves doing a self.get_value('image-tar-file')`?

Reply:
We thought it would be good to put the full dict of StepResults into extraData in the Rekor entry, and it would be easier to add image-tar-hash to the output and use [__get_all_step_results_dict()](https://github.com/ploigos/ploigos-step-runner/blob/b53a991b19f14c441622a5c8a6725e7bffd4579d/src/ploigos_step_runner/workflow_result.py#L216) since it was already implemented

Reply:
k